front/package-lock.json

```diff
@@ -210,7 +210,6 @@
       "version": "2.1.2",
       "resolved": "https://registry.npmjs.org/@ampproject/remapping/-/remapping-2.1.2.tgz",
       "integrity": "sha512-hoyByceqwKirw7w3Z7gnIIZC3Wx3J484Y3L/cMpXFbr7d9ZQj2mODrirNzcJa+SM3UlpWXYvKV4RlRpFXlWgXg==",
-      "dev": true,
```

Comment:
Once again, I picked up lots of changes to this file after merging master into the feature branch (from which this branch is based) and running `npm i`